Game/Src/Characters/Character.tscn

```diff
@@ -54,3 +58,14 @@ anims/Frame = SubResource( 1 )
 anims/RESET = SubResource( 2 )
 
 [node name="AnimationTree" type="AnimationTree" parent="."]
+
+[node name="PlayerStats" parent="." instance=ExtResource( 5 )]
+
+[node name="HealthUI" parent="." instance=ExtResource( 6 )]
+margin_left = -53.0
+margin_top = -167.0
+margin_right = -13.0
+margin_bottom = -127.0
```

Comment:
These margins looks similar to the numbers used for the sword position on top (margin_left and margin_top). Should they be defined at the beginning of the file and used on line 51?
Also you don't use them here? So I guess it is to import somewhere else?

Reply:
I have no clue, this is godot's internal mumbo jumbo with the scenes. I guess this could be done but it should not matter much